card/mod/account/spec/set/type/signup_spec.rb

```diff
@@ -205,6 +205,10 @@
       expect(@account.token).to be_present
       expect(@account.password).not_to be_present
     end
+
+    it "considers signups created by signed-in users to be invitations" do
+      @signup.format.invitation? == true
```

Comment:
True, though it isn't totally worthless; it still failed with an error before my change ;)

Will fix.